src/game/runners/ConsoleRunner.java

```diff
@@ -178,15 +185,15 @@ private void playTurn() {
 
         // Making attack to the request point.
         if (this.game.playAttack(fireToPoint)) {
-            System.out.println("NICE JOB! you successfully hit a ship.");
+            System.out.println("NICE JOB! you successfully hit a ship.\n");
         }
 
         this.showGameStatus();
     }
 
     private void showStatistics() {
         if (this.game == null) {
-            System.out.println("Statistics are available when game started.");
+            System.out.println("Statistics are available when game started.\n");
```

Comment:
Why did you add all those "\n" ? println suppose to break line after the print. Does the cms did not print new lines after it ?

Reply:
@ashavit

Reply:
@yuri1992 It does, I just wanted to add an empty line so it will be more clear when printing